ex03.ex

```diff
@@ -101,7 +107,10 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
+   def list_equal([],[]), do: true
+   def list_equal([_h | tail1],[_h | tail2]), do: list_equal(tail1, tail2)
```

Comment:
## -2

There's a bug here—the last clause means that you'll get results such as

~~~ elixir
list_equal([1,2], :wombat) #=> false
~~~

But the function should only be defined if it is passed two lists.